lib/locales/es.yml

```diff
@@ -1,27 +1,92 @@
 ---
 es:
   expectation_DoesTypeTest: '%{binding} hace comparaciones de strings innecesarias'
```

Comment:
Really, this should be something like `hace chequeos explícitos de tipo` or `pregunta por el tipo de una clase` or something like that. The fact that you're comparing strings doesn't say much about the smell.